BootloaderCorePkg/Include/HashStore.h

```diff
@@ -22,6 +22,7 @@ typedef struct {
   UINT32             Signature;
   UINT32             Valid;
   HASH_STORE_DATA    Data[HASH_INDEX_MAX_NUM];
+  UINT8              HashType;
 } HASH_STORE_TABLE;
```

Comment:
Do we need versioning for this data structure? Would we be able to update it in future to accomodate any new requirements?

Reply:
agree.  captured in previous comment.